### DISCUSSION TITLE
How to disable dark mode properly?

### DISCUSSION THREAD

**Discussion by apcamargo** [pradyunsg/furo#623] [Questions] [answered]
I'm preparing a documentation with a lot of images and I won't be able to produce alternative versions for a dark background. To solve this, I'm disabling dark mode by injecting custom CSS and JS:

```css
.theme-toggle-container {
     display: none;
}
```

```js
document.body.setAttribute("data-theme", "light");
```

The problem is that the custom JS is executed after the "default" code, so the website always blinks dark before turning light. This is not a huge issue, but can be annoying. Is it possible to have my JS code to take precedence over Sphinx/Furo's JS?

**Comment by pradyunsg** [comment, marked as answer]
See https://github.com/pradyunsg/furo/issues/100

> Is it possible to have my JS code to take precedence over Sphinx/Furo's JS?

Where are you injecting this? Furo's JS logic is a render-blocking blurb, and you'd need to have your own render-blocking blurb (or remove Furo's defaults) to achieve this.

**Reply by apcamargo** [reply]
I'm just using:

```python
html_css_files = ["css/custom.css"]
html_js_files = ["js/custom.js"]
```

I actually got the JS code from https://github.com/pradyunsg/furo/issues/28
